### PR TITLE
wails: add default layout

### PR DESCRIPTION
**Pull request by maxime-carabina**
We are adding a default layout for wails front-end screens.

### CODE REVIEW

File: wails-frontend/src/layouts/DefaultLayout/DefaultLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from 'react';
+
+interface LayoutProps {
+  children: ReactNode;
+}
+
+export function DefaultLayout(props: LayoutProps) {
+  return <div className={`bg-primary min-h-screen`}>{props.children}</div>;
```
Comment (pivilartisant) [inline wails-frontend/src/layouts/DefaultLayout/DefaultLayout.tsx:8]: I would suggest this for the default layout... that way you can just add your children in without any worry of size... and we can keep this as a standard through out the wails screens

<div className="bg-primary flex flex-col justify-center items-center h-screen w-full">
      <div className="w-1/4  max-w-xs  min-w-fit">
      </div>
</div>

Comment (pivilartisant) [reply wails-frontend/src/layouts/DefaultLayout/DefaultLayout.tsx:8]: '' <div className="bg-primary flex flex-col justify-center items-center h-screen w-full">
      <div className="w-1/4  max-w-xs  min-w-fit">
      </div>
</div> '''

Comment (pivilartisant) [inline wails-frontend/src/layouts/DefaultLayout/DefaultLayout.tsx:8]: I would suggest this for the default layout... that way you can just add your children in without any worry of size... and we can keep this as a standard through out the wails screens


```html
<div className="bg-primary flex flex-col justify-center items-center h-screen w-full">
      <div className="w-1/4  max-w-xs  min-w-fit">
      </div>
</div>
```

Comment (maxime-carabina) [reply wails-frontend/src/layouts/DefaultLayout/DefaultLayout.tsx]: I changed this! @pivilartisant 
I followed the recommendation @mazmassa gave me on Friday for the classNames.

**Review by pivilartisant — APPROVED** [review]

**Review by Thykof — APPROVED** [review]

**Review by mazmassa — APPROVED** [review]
Not sure about the name, maybe keep only layout, but that's beautiful! Thanks for that Max!

**Pull request merged**